sources/packages/web/src/router/index.ts

```diff
@@ -3,14 +3,23 @@ import { studentRoutes } from "./StudentRoutes";
 import { institutionRoutes } from "./InstitutionRoutes";
 import { aestRoutes } from "./AESTRoutes";
 import { sharedRoutes } from "./SharedRoutes";
-import {
-  InstitutionRoutesConst,
-  SharedRouteConst,
-} from "../constants/routes/RouteConstants";
-import { UserAuthorizationService } from "@/services/UserAuthorizationService";
-import { AuthService } from "@/services/AuthService";
 import { ClientIdType } from "../types/contracts/ConfigContract";
 import { supportingUsersRoutes } from "./SupportingUserRoutes";
+import { validateInstitutionUserAccess } from "./InstitutionRouteHelper";
+import { InstitutionUserTypes } from "@/types";
+
+declare module "vue-router" {
```

Comment:
for my info, why declare a module here

Reply:
Just blindly followed the way how it was given in vue documentation.

![image](https://user-images.githubusercontent.com/54600590/233191625-bf50b795-9f20-40cb-bd93-f106c2387cd6.png)
https://router.vuejs.org/guide/advanced/meta.html